osutil/epoll/epoll.go

```diff
@@ -8,15 +8,19 @@
 package epoll
```

Comment:
I noticed the blank line before this, it is wrong as it stops the comment before to show in "go doc"

Reply:
no, see new comment